webapp/src/ts/services/telemetry.service.ts

```diff
@@ -345,10 +356,8 @@ export class TelemetryService {
     databaseNames?.forEach(dbName => {
       const nameNoPrefix = dbName?.replace(this.POUCH_PREFIX, '') || '';
 
-      // Skips new Telemetry DB, then matches the old deprecated Telemetry DB.
-      if (!nameNoPrefix.startsWith(this.TELEMETRY_PREFIX)
-        && nameNoPrefix.includes(this.TELEMETRY_PREFIX)
-        && nameNoPrefix.includes(this.sessionService.userCtx().name)) {
```

Comment:
Why did you remove the check for `userCtx`? Without this won't we end up deleting someone elses telemetry on phones that are shared between users?

Reply:
ah shared phones, okay putting it back